### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Dmytro6093**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,8 +1,42 @@
...
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: int, clean_power: int,
+                 average_rating: (int, float), count_of_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, list_of_cars: list[Car]) -> object:
```
Comment (ArsenPidhoretskyi) [inline app/main.py:18]: list_of_cars -> cars
Also it returns not object. It returns float

File: app/main.py [hunk 2]
```diff
@@ -1,8 +1,42 @@
...
+        serving_price = 0
+        for i in list_of_cars:
```
Comment (ArsenPidhoretskyi) [inline app/main.py:20]: i is not good name. Rename it

File: app/main.py [hunk 3]
```diff
@@ -1,8 +1,42 @@
...
+            serving_price += self.calculate_washing_price(i)
+            self.wash_single_car(i)
+        return serving_price
+
+    def calculate_washing_price(self, veh: Car) -> float:
+        if self.clean_power > veh.clean_mark:
+            price = veh.comfort_class * (self.clean_power - veh.clean_mark) * \
```
Comment (ArsenPidhoretskyi) [inline app/main.py:27]: veh -> car

File: app/main.py [hunk 4]
```diff
@@ -1,8 +1,42 @@
...
+                (self.average_rating / self.distance_from_city_center)
+        else:
+            return 0
+        return round(price, 1)
+
+    def wash_single_car(self, veh: Car) -> None:
```
Comment (ArsenPidhoretskyi) [inline app/main.py:33]: veh -> car

**Review by ArsenPidhoretskyi — CHANGES_REQUESTED** [review]

File: app/main.py [hunk 5]
```diff
@@ -1,8 +1,42 @@
...
+
+    def serve_cars(self, cars: list[Car]) -> float:
+        serving_price = 0
+        for car in cars:
+            serving_price += self.calculate_washing_price(car)
+            self.wash_single_car(car)
+        return serving_price
+
+    def calculate_washing_price(self, car: Car) -> float:
+        if self.clean_power > car.clean_mark:
```
Comment (vasylhnatiuk) [inline app/main.py:26]: you don't need it here

Comment (vasylhnatiuk) [inline app/main.py:26]:
```suggestion
            price = car.comfort_class * (self.clean_power - car.clean_mark) * \
                (self.average_rating / self.distance_from_city_center)
             return round(price, 1)
```

Comment (Dmytro6093) [reply app/main.py]: I'm sorry, I don't understand what exactly I don't need here

**Pull request closed without merging**